src/Microsoft.VisualStudio.Editors/PropPages/AdvBuildSettingsPropPage.Designer.vb

```diff
@@ -92,7 +92,7 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
             resources.ApplyResources(Me.cboDebugInfo, "cboDebugInfo")
             Me.cboDebugInfo.DropDownStyle = System.Windows.Forms.ComboBoxStyle.DropDownList
             Me.cboDebugInfo.FormattingEnabled = True
-            Me.cboDebugInfo.Items.AddRange(New Object() {resources.GetString("cboDebugInfo.Items"), resources.GetString("cboDebugInfo.Items1"), resources.GetString("cboDebugInfo.Items2")})
+            Me.cboDebugInfo.Items.AddRange(New Object() {resources.GetString("cboDebugInfo.Items"), resources.GetString("cboDebugInfo.Items1"), resources.GetString("cboDebugInfo.Items2"), resources.GetString("cboDebugInfo.Items3"), resources.GetString("cboDebugInfo.Items4")})
```

Comment:
This is the fix for #415 - It will make it so that Portable and Compact show up in the dialog and according to the discussion in there those should show up for all projects not just .Net Core.